app/controllers/api/v0/articles_controller.rb

```diff
@@ -65,6 +65,11 @@ def update
         render "show", status: :ok
       end
 
+      def me
+        # Is pagination required?
```

Comment:
Yes, it's always a good idea :-)

You never know how many articles a person might have in the future.

Reply:
implemented!